dags/aps/utils.py

```diff
@@ -3,20 +3,26 @@
 from io import BytesIO
 
 from airflow.api.common import trigger_dag
-from airflow.models import Variable
 
 
-def set_APS_harvesting_interval(repo):
-    today = date.today()
-    aps_fetching_from_date = repo.find_the_last_uploaded_file_date()
-    aps_fetching_until_date = today.strftime("%Y-%m-%d")
-    Variable.set(
-        "aps_fetching_from_date", aps_fetching_from_date or aps_fetching_until_date
+def set_APS_harvesting_interval(repo, **kwargs):
```

Comment:
There are other publishers that  pull from API right ? If that's the case I think this function could go to common folder. What do you think ?

Reply:
Good idea, moved it